xds/internal/xdsclient/transport/lrs/lrs_stream.go

```diff
@@ -242,7 +242,7 @@ func (lrs *StreamImpl) recvFirstLoadStatsResponse(stream transport.StreamingCall
 	}
 
 	internal := resp.GetLoadReportingInterval()
-	if internal.CheckValid() != nil {
+	if err:= internal.CheckValid(); err != nil {
```

Comment:
The `vet` check runs `gofmt` and is failing because a space is needed b/w `err:=`. Can you please fix the formatting?